homeassistant/components/nina/translations/en.json

```
@@ -0,0 +1,27 @@
+{
+    "config": {
+        "abort": {
+            "single_instance_allowed": "Already configured. Only a single configuration possible."
+        },
+        "error": {
+            "cannot_connect": "Failed to connect",
+            "no_selection": "Please select at least one city/county",
+            "unknown": "Unexpected error"
+        },
+        "step": {
+            "user": {
+                "data": {
+                    "corona_filter": "Remove Corona Warnings",
+                    "regions1": "city/county (A-D)",
```

Consider starting with a capital to stay consistent

```suggestion
                    "regions1": "City/County (A-D)",
                    ...
```